_global/js/card.js

```diff
@@ -1,5 +1,14 @@
 var quickCards = document.querySelectorAll('quick-card');
 
+window.onload = function startMain(){
```

Comment:
Maybe use addEventListener like suggested here? https://stackoverflow.com/questions/16463998/multiple-javascript-window-onload-solution

That way it's possible to have an onload function in multiple script files and include all of them.

Reply:
Sounds like a good solution. I was just following along with what was done in board_front.js

Reply:
Fair enough. Feel free to leave this as-is then, and after you're done I'll sent a follow-up pull request to change both instances into addEventListener.

Reply:
Great. I wasn't sure how to do what you were asking myself, which is why I didn't take it on.